src/common/catalog-entities/kubernetes-cluster.ts

```diff
@@ -134,7 +135,7 @@ class KubernetesClusterCategory extends CatalogCategory {
   public readonly kind = "CatalogCategory";
   public metadata = {
     name: "Clusters",
-    icon: require("./icons/kubernetes.svg?raw"),
+    icon: KubeClusterCategoryIcon,
```

Comment:
Woot! 🎉

Reply:
By default imported `*.svg` files contain data-url (string) that can be consumed normally by `<img src>` or with `<Icon svg`/>` as inline/raw xml icon.

Reply:
Amazing!